### ISSUE TITLE
Landmine notification

### ISSUE THREAD

**Issue by samdabloxyccola** [DankMemer/dank-memer-suggestions-and-bugs#715]
So, when you try to rob someone and you die because of their landmine, they don't receive on a notification on who died due to their landmine. 

Solution: Add a notification so that when someone triggers your landmine, you receive a notification.

E.g: BillyJones#0001 tried to rob DwayneTheRockJohnson#0503. However, the landmine set by Dwayne got triggered by Billy, and Billy died. 2 hours later, when Dwayne came back from grocery shopping, he found his landmine missing (Given that he had planted the landmine on the same day). He checked his notifications, however, there was no notification on who attempted a rob on him. 

The notification should go like this: BillyJones#0001 attempted to rob you in <Server name>, but was stopped/killed by your landmine!

**Comment by DottySpam** [comment]
Yes so true I get fed up when I don't even know who attempted to rob me
